### ISSUE TITLE
Cannot launch the game

### ISSUE THREAD

**Issue by atmotrix** [Regalis11/Barotrauma#10522]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So my friend just bought Barotrauma on team and wanted to to play, but after launching game this window appears.
![image](https://user-images.githubusercontent.com/118890590/203514401-9bdc1920-c905-4d5a-8283-873423ed8ce9.png)

### Reproduction steps

1. Launch Game
2. Small Barotrauma window appears
3. Screenshot attached

### Bug prevalence

Happens every time I play

### Version

0.19.14.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Barotrauma Client crash report (generated on 11/23/2022 15:24:58)


Barotrauma seems to have crashed. Sorry for the inconvenience!


0DC439F0B0CA5A5A6693500952309D61


Game version 0.19.14.0 (ReleaseWindows, branch release, revision 0daf5b87bd)
Graphics mode: 1366x768 (BorderlessWindowed)
VSync ON
Language: English
Selected content packages: None
Level seed: no level loaded
Loaded submarine: None
Selected screen: None
SteamManager initialized


System info:
Operating system: Microsoft Windows NT 6.3.9600.0 64 bit
GPU name: NVIDIA GeForce 840M
Display mode: {Width:1366 Height:768 Format:Color AspectRatio:1.7786459}
GPU status: Normal


Exception: HRESULT: [0x887A0005], Module: [SharpDX.DXGI], ApiCode: [DXGI_ERROR_DEVICE_REMOVED/DeviceRemoved], Message: The GPU device instance is suspended. To determine the appropriate action, use GetDeviceRemovedReason.
(SharpDX.SharpDXException)
Device removed reason: HRESULT = 0x887A0020
Target site: Void CheckError()
Stack trace:
at SharpDX.Result.CheckError()
at SharpDX.DXGI.SwapChain.Present(Int32 syncInterval, PresentFlags flags)
at Microsoft.Xna.Framework.Graphics.GraphicsDevice.PlatformPresent() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Graphics\GraphicsDevice.DirectX.cs:line 1010
at Microsoft.Xna.Framework.Graphics.GraphicsDevice.Present() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Graphics\GraphicsDevice.cs:line 600
at Microsoft.Xna.Framework.SdlGamePlatform.Present() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\SDL\SDLGamePlatform.cs:line 250
at Microsoft.Xna.Framework.Game.EndDraw() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 522
at Microsoft.Xna.Framework.Game.DoDraw(GameTime gameTime) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 667
at Microsoft.Xna.Framework.Game.Tick() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 508
at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\SDL\SDLGamePlatform.cs:line 92
at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 397
at Microsoft.Xna.Framework.Game.Run() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 367
at Barotrauma.Program.Main(String[] args) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\Program.cs:line 58


Last debug messages:
[11/23/2022 15:24:58] Attempting to open ALC device "OpenAL Soft on Speakers (Realtek High Definition Audio)"
```

**Comment by atmotrix** [comment]
I hope I wrote enough info for you to understand that situation. Thank you very much for looking into!

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/1833

There are some potential workarounds to the crash listed in that ticket, please take a look and don't hesitate to comment there if none of those help. Sorry for the inconvenience!

**Comment by atmotrix** [comment]
> Duplicate of #1833
> 
> There are some potential workarounds to the crash listed in that ticket, please take a look and don't hesitate to comment there if none of those help. Sorry for the inconvenience!

Yes, I am sorry for making duplicate of another bug report. I've read all potential fixes but none of them worked. Also, I've discovered feature in config_player.xml. There were a few problems with my resolution: width and height had values "0" and i changed them to proper values. Still no effect on game though. I've tried to download older drivers. checked right version of DirectX, disabled image sharpening and even reinstalled the game. None of that worked. Maybe I missed some of the crucial yet useful hints on that situation?
